src/packages/vue/index.md

```diff
@@ -92,4 +92,35 @@ class ApplicationUsingReactWindow extends Window {
 }
 ```
 
+If you want to have events between VueJS instances that reside in *separate* OSjs packages/applications then...
+In the "broadcasting" VueJS instance, you will need to include the OSjs application `app` instace (main.js):
+```
```

Comment:
Might be better as: `If you want to communicate between Vue components in *separate* OS.js packages, you can use Process broadcasting. To expose the OS.js application instance to your Vue component you can bind it in the data of the  root Vue instance:`.